tests/test_examples.py

```diff
@@ -19,6 +19,7 @@
     ['contrib/autoname/mixture.py', '--num-epochs=1'],
     ['contrib/autoname/tree_data.py', '--num-epochs=1'],
     ['contrib/gp/sv-dkl.py', '--epochs=1', '--num-inducing=4'],
+    ['contrib/oed/ab_test.py'],
```

Comment:
Maybe add a command line option to control `num_steps` in `ab_test.py` so you can run the example test for less time?